src/Network.cpp

```diff
@@ -3,49 +3,49 @@
 namespace Network {
   byte mode;
 
-  void init(String ssid, String pwd) {
+  void init() {
     mode = MODES::DISCONNECTED;
     WiFi.setSleepMode(WIFI_NONE_SLEEP);
 
-    if (beginST(ssid.c_str(), pwd.c_str())) {
+    if (beginST()) {
```

Comment:
The AP mode is never being execute like it is now

Reply:
The reason behind this is because we cannot tell if the connection can be achieved just from the settings, we neeed to try them. And default values could be valid

Reply:
I did test it and it's works! the initial AP is starting well.. I don't understand why though.. On the code looks like it will never start. I'm missing something on the libs functions